packages/esm-outpatient-app/src/past-visit/past-visit-details/vitals.component.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { Grid, Row, Tile } from 'carbon-components-react';
+import { useTranslation } from 'react-i18next';
+import { calculateBMI, assessValue, getReferenceRangesForConcept } from '../../current-visit/current-visit.resource';
+import { PatientVitals } from '../../types/index';
+import { useConfig } from '@openmrs/esm-framework';
+import styles from './past-visit-summary.scss';
+import CircleFillGlyph from '@carbon/icons-react/es/circle--solid/16';
+import { useVitalsConceptMetadata } from '../../current-visit/hooks/useVitalsConceptMetadata';
+import { ConfigObject } from '../../config-schema';
+
+interface VitalsComponentProps {
+  vitals: Array<PatientVitals>;
+}
+
+const Vitals: React.FC<VitalsComponentProps> = ({ vitals }) => {
```

Comment:
This file for vitals component is duplicated

Reply:
@makombe im now passing a prop to the current visit vitals so that i can reuse it